src/gc/gcinterface.dac.h

```diff
@@ -0,0 +1,144 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef _GC_INTERFACE_DAC_H_
+#define _GC_INTERFACE_DAC_H_
+
+// This file defines the interface between the GC and the DAC. The interface consists of two things:
+//   1. A number of variables ("DAC vars") whose addresses are exposed to the DAC (see "struct GcDacVars")
+//   2. A number of types that are analogues to GC-internal types. These types expose a subset of the
+//      GC-internal type's fields, while still maintaining the same layout.
+// This interface is strictly versioned, see gcinterface.dacvars.def for more information.
+
+#define NUM_GC_DATA_POINTS             9
```

Comment:
@noahfalk So, unfortunately, these values influence the layout of `DacpGCInterestingInfoData`: https://github.com/dotnet/coreclr/blob/master/src/inc/dacprivate.h#L787.